### ISSUE TITLE
Robigalia: developing device drivers and other operating system services in pure Rust, using the seL4 microkernel

### ISSUE THREAD

**Issue by emberian** [RustSydney/talks#12]
I can give this, although no earlier than February.

**Comment by yoshuawuyts** [comment]
that sounds heaps interesting!

**Comment by caspark** [comment]
Excellent, Feb it is (or later). I'll get back to you on the exact date.

**Comment by caspark** [comment]
@cmr would you like to give this talk for Meetup.8 on Tuesday February 16th?

**Comment by emberian** [comment]
No, I'd prefer more time for the project to mature, and to prepare the talk. 

On January 12, 2016 6:50:22 PM EST, Caspar Krieger notifications@github.com wrote:

> @cmr would you like to give this talk for Meetup.8 on Tuesday February
> 16th?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/RustSydney/talks/issues/12#issuecomment-171102778

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by khklau** [comment]
@cmr are you back from the APSys 2016 conference? Are you able to give the talk now?
